crates/history/Cargo.toml

```
@@ -24,6 +24,9 @@ thiserror = { version = "1.0", optional = true }
 version = "0.3"
 features = ["History", "Window", "Location", "Url"]
 
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+getrandom = {version="0.2.10", features = ["js"]}
```

```suggestion
getrandom = { version = "0.2.10", features = ["js"] }
```

Should have spacing like other dependencies.

---

I'd love it when cargo fmt would format also this. Fixed.